deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir["#{options[:metadata_path]}/app_icon.{png,jpg}"].first
```

Comment:
@hjanuschka why would the user download over his existing metadata? This is a weird scenario and in my head, unsupported. This use case is the same as let's say download over an older version of the app metadata. It can lead to weird cases.